packages/react-native-reanimated/src/publicGlobals.ts

```diff
@@ -21,4 +23,9 @@ declare global {
    * which is the Reanimated UI runtime.
    */
   var _WORKLET_RUNTIME: ArrayBuffer;
+
+  /**
+   * This is a custom error class that is thrown by Reanimated.
+   */
+  var ReanimatedError: typeof ReanimatedErrorClass;
```

Comment:
I don't think we should make it a public global. Let's move it to `globals.d.ts` to make it private instead.